client/src/components/AddMemoryForm.tsx

```diff
@@ -1,130 +1,125 @@
 import * as React from "react";
-
 import { Button } from "@chakra-ui/react";
 import { userData } from "./main/side.nav";
 import { personData } from "./person.details";
-import { useQuery } from "react-query";
-
 import {
-  Menu,
-  MenuButton,
-  MenuList,
-  MenuItem,
+    Menu,
+    MenuButton,
+    MenuList,
+    MenuItem,
 } from "@chakra-ui/react";
 
 export default function AddMemoryForm() {
-  const [title, setTitle] = React.useState("");
-  const [text, setText] = React.useState("");
-  const [person, setPerson] = React.useState<string[]>([]);
-  const [memoryCount, setMemoryCount] = React.useState("0");
-  const [user, setUser] = React.useState<null | userData>();
-  const userInput: { title: string; text: string; person: string[] } = {
-    title: title,
-    text: text,
-    person: person,
-  };
+    const [title, setTitle] = React.useState("");
+    const [text, setText] = React.useState("");
+    const [person, setPerson] = React.useState<string[]>([]);
```

Comment:
I think this is plural: should it not be `people`, and `setPeople`?

Reply:
I checked, single person is correct here - we are setting the ONE person this memory belongs to